packages/transition-backend/src/services/transitRouting/TrAccessibilityMapBatch.ts

```diff
@@ -205,7 +197,7 @@ export const batchAccessibilityMap = async (
         await promiseQueue.onIdle();
 
         progressEmitter.emit('progress', { name: 'StoppingRoutingParallelServers', progress: 0.0 });
-        const stopStatus = await TrRoutingProcessManager.stopMultiple(trRoutingInstancesCount);
+        const stopStatus = await TrRoutingProcessManager.stopBatch();
```

Comment:
Shouldn't you give the port number here? In case we have concurrency of jobs at some point and have multiple instances running.